### PR TITLE
[ENDPOINT] UPDATE CATEGORIES OT-198-44

### PR DESCRIPTION
**Pull request by JoaquinBar1**
[OT-198-44](https://alkemy-labs.atlassian.net/jira/software/c/projects/OT198/boards/278?modal=detail&selectedIssue=OT198-44)

### Description

AS admin user
I WANT to modify an existing category
TO keep the information updated

Criteria of acceptance:
PUT /categories/:id - It must validate that the category exists and update it, otherwise return an error

## Evidence 

STATUS OK

PUT - STATUS OK

NOT FOUND

PUT NOT FOUND

### CODE REVIEW

**Review by nicobes14 — APPROVED** [review]
todo ok!

File: services/categories.js [hunk 1]
```diff
@@ -27,6 +27,17 @@ const listCategories = async () => {
   }
 }
 
+const upDateCategory = async (categoryToEdit, id) => {
+  try {
+    const editCategory = await Category.update(categoryToEdit, {
+      where: { id },
+    })
+    return editCategory[0] === 1 ? { code: 200, status: true, message: 'Category edited' } : { code: 400, status: false, message: `Category ${id} not found` }
```
Comment (HernanGarcia1985) [inline services/categories.js:35]: Un detalle debería ser código 404 Not found para normalizar con otras respuestas.

**Review by HernanGarcia1985 — APPROVED** [review]
Todo OK!!!

**Review by maxisosa89 — APPROVED** [review]
Mismo comentario que Hernan, el resto me parece bien.

**Pull request merged**